app/views/users/backup_code_setup/confirm_backup_codes.html.erb

```diff
@@ -0,0 +1,37 @@
+<% title t('titles.backup_codes') %>
+
+<%= render PageHeadingComponent.new.with_content(t('titles.backup_codes')) %>
+
+<p>
+  <%= t('two_factor_authentication.backup_codes.warning_html') %>
+</p>
+
+<p>
+  <%= t('two_factor_authentication.backup_codes.instructions', app_name: APP_NAME) %>
+</p>
+
+<div class="grid-row">
```

Comment:
There should be a bit more margin between this and the preceding paragraph (standard 2.5rem above primary button).

```suggestion
<div class="grid-row margin-top-5">
```